music21/harmony.py

```diff
@@ -580,11 +598,11 @@ class ChordStepModification(prebase.ProtoM21Object):
 
     # INITIALIZER #
 
-    def __init__(self, modType=None, degree=None, intervalObj=None):
-        self._modType = None  # add, alter, subtract
-        self._interval = None  # alteration of degree, alter ints in mxl
-        self._degree = None  # the degree number, where 3 is the third
-        # use properties if defined
+    def __init__(self, modType=None, degree=None, intervalObj=interval.Interval('P1')):
+        self._modType: str | None = None  # add, alter, subtract
```

Comment:
I wonder if we can use `''` for None going forward.  There were a lot of str | None in the code before typing became a thing but now it's biting us.